spark-paper/src/main/java/me/lucko/spark/paper/PaperCommandSender.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of spark.
+ *
+ *  Copyright (c) lucko (Luck) <luck@lucko.me>
+ *  Copyright (c) contributors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package me.lucko.spark.paper;
+
+import me.lucko.spark.common.command.sender.AbstractCommandSender;
+import net.kyori.adventure.text.Component;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.UUID;
+
+public class PaperCommandSender extends AbstractCommandSender<CommandSender> {
+
+    public PaperCommandSender(CommandSender sender) {
+        super(sender);
+    }
+
+    @Override
+    public String getName() {
+        return this.delegate.getName();
+    }
+
+    @Override
+    public UUID getUniqueId() {
+        if (super.delegate instanceof Player) {
+            return ((Player) super.delegate).getUniqueId();
```

Comment:
```suggestion
        if (super.delegate instanceof Player player) {
            return player.getUniqueId();
```
small nitpick, since this module targets java 21 we can make this look nicer

Reply:
nice :) thanks!